build.sbt

```diff
@@ -41,6 +43,7 @@ libraryDependencies ++= Seq(
   "mysql" % "mysql-connector-java" % "5.1.35",
   "org.liquibase" % "liquibase-core" % "3.3.5",
   "org.hsqldb" % "hsqldb" % "2.3.2",
+  "com.sendgrid" % "sendgrid-java" % "2.2.2",
```

Comment:
the next version of the library is 3.0.0, which came out about a week ago. it's currently in beta, though... so i don't know if we want to use it yet. it is a breaking change for the library